Themes/default/scripts/script.js

```diff
@@ -338,11 +338,16 @@ function reqOverlayDiv(desktopURL, sHeader, sIcon)
 {
 	// Set up our div details
 	var sAjax_indicator = '<div class="centertext"><img src="' + smf_images_url + '/loading_sm.gif"></div>';
-	var sIcon = smf_images_url + '/' + (typeof(sIcon) == 'string' ? sIcon : 'helptopics.png');
 	var sHeader = typeof(sHeader) == 'string' ? sHeader : help_popup_heading_text;
 
+	var containerOptions;
+	if (typeof(sIcon) == 'string' && sIcon.match(/\.(gif|png|jpe?g|svg|bmp|tiff)$/) != null)
```

Comment:
Where does it look for a hard coded icon with this change?
e.g. Before I could do something like this:
`reqOverlayDiv(this.href, \'%2$s\', \'/icons/modify_inline.png\')`
And use the modify_inline.png icon (for example), with the images_url gone, where is it looking for this icon if I were to do the same?

Reply:
Oops. That was a silly mistake. I will fix that soon.

Reply:
Fixed in b311437

Reply:
It's working now, thanks.